CHANGELOG-DEV.md

```diff
@@ -5,3 +5,7 @@ HumHub Changelog
 -------------------
 - Fix #6216: Spaces icon in admin menu
 - Fix #6229: Bug on saving forms: Zend OPcache API is restricted by "restrict_api"
+
+1.14.0 (Unreleased)
+-------------------
+- Enh #6236: Logging: Show log entries from migrations with category migration
```

Comment:
Please add into `v1.15` section

Reply:
Ok, so it won't make it into v1.14 then, I guess?

Hence I will also update the respective tags to `@since 1.15.0`?

Reply:
Oh yes, update the `@since` tags would be good. `v1.14` will hopefully be released as a stable this week. `v1.15` is planned for Juny/July.